packages/grid/test/keyboard-navigation.test.js

```diff
@@ -18,6 +18,7 @@ import '../vaadin-grid.js';
 import '../vaadin-grid-tree-column.js';
 import '../vaadin-grid-column-group.js';
 import '../vaadin-grid-selection-column.js';
+import { isChrome } from '@vaadin/component-base/src/browser-utils.js';
```

Comment:
Not sure if that import is good practice in tests. Can replace or inline it if necessary.

Reply:
You can use the helper from `@vaadin/testing-helpers`:

```suggestion
impory { isChrome } from '@vaadin/testing-helpers';
```